posts/2019-01-17-Rust-1.32.0.md

```diff
@@ -265,16 +264,15 @@ primitives now provide conversion functions to and from byte-arrays with
 specified endianness. These six functions are named `to_<endian>_bytes` and
 `from_<endian>_bytes`, where `<endian>` is one of:
 
-* `ne` - native endianness
-* `le` - little endian
-* `be` - big endian
+- `ne` - native endianness
+- `le` - little endian
+- `be` - big endian
 
 See the [detailed release notes][notes] for more details.
 
 ### Cargo features
 
-Cargo gained [`cargo c` as an alias for `cargo
-check`](https://github.com/rust-lang/cargo/pull/6218/), and now [allows
+Cargo gained [`cargo c` as an alias for `cargo check`](https://github.com/rust-lang/cargo/pull/6218/), and now [allows
```

Comment:
If you could split the post edits into a separate PR that'd be great!

Reply:
yes, big 👍  here